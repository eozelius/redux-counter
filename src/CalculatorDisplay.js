import React from 'react'

const CalculatorDisplay = (props) => {
  return (
    <div className='displayContainer'>
      <h1>{props.number}</h1>
    </div>
  )
}

export default CalculatorDisplay
