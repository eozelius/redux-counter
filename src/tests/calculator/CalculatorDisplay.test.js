import React from 'react'
import CalculatorDisplay from '../../CalculatorDisplay'

describe(CalculatorDisplay, () => {
  it('mounts properly', () => {
    const display = shallow(<CalculatorDisplay />)
    expect(display).toMatchSnapshot()
  })
})
