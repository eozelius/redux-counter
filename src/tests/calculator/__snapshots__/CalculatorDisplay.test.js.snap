// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorDisplay mounts properly 1`] = `
<div
  className="displayContainer"
>
  <h1 />
</div>
`;
